src/main/java/uk/gov/register/presentation/resource/DataResource.java

```diff
@@ -50,19 +51,21 @@ public Response downloadTorrent() {
     @GET
     @Path("/feed")
     @Produces({MediaType.TEXT_HTML, MediaType.APPLICATION_JSON, ExtraMediaType.TEXT_YAML, ExtraMediaType.TEXT_CSV, ExtraMediaType.TEXT_TSV, ExtraMediaType.TEXT_TTL})
-    public EntryListView feed() {
-        return viewFactory.getEntryFeedView(
-                queryDAO.getAllEntries(ENTRY_LIMIT)
-        );
+    public EntryListView feed(@QueryParam("pageIndex") long pageIndex, @QueryParam("pageSize") long pageSize) {
```

Comment:
what values will these `long`s be given if the query string isn't present? is it worth documenting this, or using a `Long` or `Optional<Long>` type to indicate that the values are optional?